docs/userguide/targets.md

```diff
@@ -1,18 +1,87 @@
 # Targets
 
-## General information
+Targets play a crucial role as the connection between observables measured in an
+experiment and the machine learning core behind BayBE.
+In general, it is expected that you create one [`Target`](baybe.targets.base.Target)
+object for each of your observables.
+The way BayBE treats multiple targets is then controlled via the 
+[`Objective`](../../userguide/objective).
 
-For BaybBE, targets play a crucial role in modeling the function that requires optimization during the DoE campaign.
-Targets are used to model the function that should be optimized by the DoE campaign. They are implemented via the [`Target`](baybe.targets.base.Target) class.
-Each target has a name as well as an abstract transform function. This function is used to transform data into computational representation. 
+## NumericalTarget
+Besides the `name`, a [`NumericalTarget`](baybe.targets.numerical.NumericalTarget)
+has the following attributes:
+* **The optimization** `mode`: This specifies whether we want to minimize or maximize
+  the target or whether we want to match a specific value.
+* **Bounds:** Define `bounds` that constrain the range of target values.
+* **A transformation function:** When bounds are provided, a `transformation` is
+  used to map target values onto the [0,1] interval.
 
-BayBE currently supports a single type of target, called [`NumericalTarget`](baybe.targets.numerical.NumericalTarget), specifically designed for numerical modeling.
+Below a visualization of possible choices for `transformation`, where `lower` and
+`upper` are the entries provided via `bounds`:
+![Transforms](../_static/target_transforms.svg)
 
-## Numerical targets
+### MIN and MAX mode
+Here two examples for simple maximization and minimization targets:
+```python
+from baybe.targets import NumericalTarget, TargetMode, TargetTransformation
 
-Besides their name, numerical targets have the following attributes.
-* **The optimization mode:** This mode dictates whether we want to minimize or maximize the target or whether we want to match a specific value.
-* **Bounds:** Define optional bounds that constrain the range of target values.
-* **A bound transformtaion function:** When bounds are provided, a bound transformation function is used to map target values onto the [0,1] interval. This is only applicable in minimization and maximization mode.
+max_target = NumericalTarget(
+    name="Target1",
```

Comment:
Here and in several other lines I will not mark: Would prefer underscores in front of stuff like 1, A, B and so on, but feel free to just resolve if you disagree.

Reply:
agree, in fact this is the style I've used in other pages as well not so that would be more consistent, changed